src/test/resources/application.properties

```diff
@@ -3,3 +3,4 @@ spring.datasource.driverClassName=org.h2.Driver
 spring.datasource.username=sa
 spring.datasource.password=password
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+spring.liquibase.enabled=false
```

Comment:
why do you disable liquibase?

Reply:
disable liquibase in test class because of 'liquibase referential integrity violation' when use 'mvn clean package'.  Нарушение ссылочной целостности: "FK_USER_ROLES_ROLE: PUBLIC.USER_ROLES FOREIGN KEY(ROLE_ID) REFERENCES PUBLIC.ROLES(ID) (CAST(2 AS BIGINT))"
Referential integrity constraint violation: "FK_USER_ROLES_ROLE: PUBLIC.USER_ROLES FOREIGN KEY(ROLE_ID) REFERENCES PUBLIC.ROLES(ID) (CAST(2 AS BIGINT))"; SQL statement:
INSERT INTO PUBLIC.user_roles (user_id, role_id) VALUES ('1', '2') [23506-224] [Failed SQL: (23506) INSERT INTO PUBLIC.user_roles (user_id, role_id) VALUES ('1', '2')]

Reply:
try to resolve this error withour disabling liquibase in test